### PR TITLE
add events service for buildbot statistics

### PR DESCRIPTION
**Pull request by tardyp**

### CODE REVIEW

File: templates/track-config.sh [hunk 1]
```diff
@@ -19,8 +19,8 @@ fi
 git config user.email "{{ track_config['default_author_email'] }}"
 git config user.name "{{ track_config['default_author_name'] }}"
 
-# check for changes
-if git status 2>/dev/null | grep -q 'working directory clean'; then
+# check for changes (some versions of git have different messages)
+if git status 2>/dev/null | egrep -qe 'working (tree|directory) clean'; then
```
Comment (seankelly) [inline templates/track-config.sh]: May be useful to use the `--short` option for more reliable machine parsing.

Comment (seankelly) [inline templates/track-config.sh]: Something like `git status --short | grep -q '^[MARCD ][MARCD ]'` I think.

File: jail-events.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    service_name: events
+    service_dir: "{{ getent_passwd[worker_account].4 }}/run"
+    service_command: "{{ getent_passwd[worker_account].4 }}/{{ env_name }}/bin/listen-events"
+    service_user: "{{ worker_account }}"
+  - role: nginx
+    nginx_template: proxy
+    server_name: "{{ web_host_name }}"
+    ssl: true
+    upstream_url: 192.168.80.227:8080
+    ssl: True
```
Comment (seankelly) [inline jail-events.yml]: `ssl` specified twice.

File: templates/track-config.sh (hunk 1 above)
Comment (tardyp) [inline templates/track-config.sh]: looks like there is --porcelain for resistance to change
http://stackoverflow.com/questions/5139290/how-to-check-if-theres-nothing-to-be-committed-in-the-current-branch

**Pull request merged**